labgrid/remote/exporter.py

```diff
@@ -220,15 +220,30 @@ def _start(self, start_params):
         assert self.child is None
         assert start_params['path'].startswith('/dev/')
         self.port = get_free_port()
-        self.child = subprocess.Popen([
-            self.ser2net_bin,
-            '-d',
-            '-n',
-            '-C',
-            '{}:telnet:0:{}:{} 8DATABITS NONE 1STOPBIT LOCAL'.format(
-                self.port, start_params['path'], self.local.speed
-            ),
-        ])
+
+        # Ser2net has switched to using YAML format at version 4.0.0.
+        _, _, version = str(subprocess.check_output(['ser2net','-v'])).split(' ')
+        if version > "4.0.0":
```

Comment:
we should probably parse out the major version and do the check against integers. This breaks if another `.0` is added:
```
python 3.8.3 (default, May 13 2020, 19:59:26)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> "4.0.0.0" > "4.0.0"
True
```

Reply:
Will do